test/integration/node-specific/client_close.test.ts

```diff
@@ -584,7 +699,57 @@ describe.skip('MongoClient.close() Integration', () => {
 
   describe('Server resource: Cursor', () => {
     describe('after cursors are created', () => {
-      it.skip('all active server-side cursors are closed by client.close()', async function () {});
+      let client;
+      let coll;
+      let cursor;
+      let utilClient;
+
+      beforeEach(async function () {
+        client = this.configuration.newClient();
+        utilClient = this.configuration.newClient();
+        await client.connect();
+        coll = await client.db('db').collection('coll', { capped: true, size: 1_000_000 });
```

Comment:
`{ capped: true, size: 1_000_000 }`

Can we set a `max` here as well and insert less than whatever `max` is? (so 4 currently). It makes it more clear what its capped at, 1mb is a bit harder to reason about (speaking generally, obviously these documents are less than 1mb)

Reply:
Yep, made the change!